exercises/robot-simulator/lib/position.dart

```diff
@@ -0,0 +1,11 @@
+class Position {
+  num x, y;
+
+  Position(this.x, this.y);
+
+  @override
+  bool operator ==(Object other) => other is Position && other.x == this.x && other.y == this.y;
```

Comment:
Recommend: Also override `hashCode` 
Or, If `position1 == position2` is not being used at all. Remove this override. By default dart uses `identical(position1, position2)` which suffices for example use case

Reply:
I'm using the `equals` method in the test file when I compare both positions (and I don't want to use `identical` because it's not the same object).